distribution/changelog.txt

```diff
@@ -1,5 +1,6 @@
 0.4.22 (in development)
 ------------------------------------------------------------------------
+- Feature: [#24206] Add plugin handle for breaking down and fixing rides.
```

Comment:
Looking at the previous changelog entries it usually looks like following, also it could be a bit clearer to what was added.
```
- Feature: [#20539] [Plugin] Add API for getting a ride’s statistics.
- Feature: [#21521] [Plugin] Add hook “park.guest.softcap.calculate” called before calculating the soft guest cap.
```

Reply:
better now?

Reply:
I think so.